CODEOWNERS

```diff
@@ -11,8 +11,10 @@ gen3.theanvil.io/manifest.json @GARUPP-zz
 staging.theanvil.io/manifest.json @GARUPP-zz
 internalstaging.theanvil.io/manifest.json @GARUPP-zz
 
-tbi.datacommons.io/manifest.json @gkuffel
+tbi.datacommons.io/manifest.json @dgrantkeane @GARUPP-zz
+
 ibdgc.datacommons.io/manifest.json @gkuffel
+jcoin.datacommons.io/manifest.json @gkuffel
```

Comment:
@atharvar28 , for JINK environments we have to tag both 
`@gkuffel @paulineribeyre`